vue.config.js

```diff
@@ -2,8 +2,7 @@ const { defineConfig } = require("@vue/cli-service");
 const NodePolyfillPlugin = require("node-polyfill-webpack-plugin");
 
 module.exports = defineConfig({
-  publicPath:
-    process.env.NODE_ENV === "production" ? process.env.BASE_URL : "/",
+  publicPath: process.env.BASE_URL,
```

Comment:
With this change, we'll need to update the `README.md` initial instructions for `npm run serve` to be:

```
Server will be located at http://localhost:8080/<BASE_URL from your .env>
```

Or perhaps this, since `/msc-animet` is the default BASE_URL in our env.

```
Server will be located at http://localhost:8080/msc-animet
```

Or, perhaps we simply revert? `process.env.NODE_ENV === "production" ? process.env.BASE_URL : "/",` is a nice safeguard to separate production deployment from other deployment types (ie. `development`, `testing`).